tasks/steep.rake

```diff
@@ -64,6 +64,13 @@ namespace :steep do
         $stdout.write('|')
         $stdout.write("\n")
       end
+
+      # Append ignored files from Steepfile to the end of the steep/typecheck summary
+      steepfile_path = 'Steepfile'
```

Comment:
My 50c here - this variable brings no value and could be inlined in the place of usage.